tensilelite/Tensile/SolutionStructs.py

```diff
@@ -1954,6 +1958,34 @@ def assignDerivedParameters(state):
       state["SynchronizerSizeCheck"] = 1
     #   state["BatchSizeEqual"] = 1
 
+    isa = tuple(state["ISA"])
+    
+    if state["StreamK"] != 0:
+      if state["MIWaveGroup"][0] * state["MIWaveGroup"][1] != 4:
+        reject(state, "Stream-K requries MIWaveGroup0*MIWaveGroup1=4")
+      if state["EnableMatrixInstruction"] and globalParameters["AsmCaps"][isa]["HasWMMA"]:
+        reject(state, "Stream-K untested with WMMA")
+      if state["GlobalSplitU"] > 0:
+        reject(state, "Cannot enable both Stream-K and GSU")
```

Comment:
Can we force GSU to 0?

Reply:
I could make that change. Do you think it's better to force change the setting if stream-k is enabled, rather than reject the parameter combo?

Reply:
I think so. tensilelite has more auto-set parameters compared to Tensile.
If we reject GSU>0, we need to specify GSU=0 for StreamK.

Reply:
Sounds good, posted an update